bot.ml

```diff
@@ -65,13 +65,7 @@ let git_delete repo remote_branch_name =
   git_push ~force:false repo "" remote_branch_name
 
 let git_make_ancestor ~base ref2 =
-  (* Ensure state is clean *)
-  "git rm .gitattributes && git reset --hard"
-  |&& Printf.sprintf "git checkout %s" ref2
-  |&& Printf.sprintf
-        "git merge %s -m \"Bot merge $(git rev-parse %s) into $(git rev-parse \
-         %s)\" || { git merge --abort; false; }"
-        base base ref2
+  Printf.sprintf "make_ancestor.sh %s %s" base ref2
```

Comment:
Just doing `./make_ancestor.sh` should work because all the commands are executed in the current working directory by default (this is how it's done in `backport_pr` below).

Reply:
>all the commands are executed in the current working directory by default

How does that work, isn't the working directory the git repo for coq?

Reply:
We clone coq in `./repo`. Thus most commands start with `cd_repo |&&`, including the one in which this function is called. So indeed, it should rather be:

```suggestion
  Printf.sprintf "../make_ancestor.sh %s %s" base ref2
```

One nice thing about your change is that it will allow once again considering to make the git repository bare (and only its worktrees are not). In this case, we can even put the git repository in the same location as the scripts instead of a sub-directory.

Reply:
Changed.